.circleci/config.yml

```diff
@@ -7,18 +7,63 @@ jobs:
     steps:
       - checkout
 
+      - restore_cache:
+          key: v1-py3-{{ .Branch }}-{{ checksum "setup.py" }}
+
+      - run:
+          name: Create a virtualenv
+          command: |
+            mkdir -p /tmp/venv/tutorial
+            python -m venv /tmp/venv/tutorial
+            echo "source /tmp/venv/tutorial/bin/activate" >> $BASH_ENV
+
       - run:
           name: Install dependencies
           command: make install
 
+      - save_cache:
+          key: v1-py3-{{ .Branch }}-{{ checksum "setup.py" }}
+          paths:
+            - /tmp/venv/tutorial
+
       - run:
           name: Test notebooks
           command: |
             cd notebooks
             make test
 
+  save_notebooks:
+    docker:
+      - image: python:3.7
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          key: v1-py3-{{ .Branch }}-{{ checksum "setup.py" }}
+
+      - run:
+          name: Test notebooks
+          command: |
+            cd notebooks
+            make test
+
+      - run:
+          name: Commit notebooks updated by tests
+          command: |
+            git add .
+            git commit -m "Save notebooks execution results"
+            git push
```

Comment:
I'm not sure this will work from CircleCI, but I guess there's no harm in merging to try it.

Reply:
Ok, thank you!

Additional information: `openfisca-bot` user key added to CircleCI
and `openfisca-bot` belongs to the `Core-Contrib` team that has access to `tutorial` repository.

Reply:
Aha! I guess I should have said "I'm not sure *how* this will work from CircleCI" ;)